<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1193</width>
         <height>861</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="verticalSpacing">
         <number>1</number>
        </property>
        <item row="26" column="0" colspan="3">
         <widget class="QLabel" name="label_26">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Stopper và offset vị trí</string>
          </property>
         </widget>
        </item>
        <item row="22" column="1">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Dữ liệu bar code</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Đường dẫn chương trình IV</string>
          </property>
         </widget>
        </item>
        <item row="25" column="1">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Kết quả in</string>
          </property>
         </widget>
        </item>
        <item row="23" column="1">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Barcode OK</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Mật khẩu admin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label_23">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Mật khẩu bảo vệ</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Xác nhận trigger</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QComboBox" name="cbxLaserPort">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Mật khẩu super</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Dừng</string>
          </property>
         </widget>
        </item>
        <item row="24" column="1">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Barcode NG</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Trigger</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="txtDatabasePath">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Sẵn sàng</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Độ rộng băng tải</string>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Xác nhận barcode</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Đường dẫn dữ liệu</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Cổng</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Stopper</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Cổng</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Xác nhận dừng</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Đường dẫn dữ liệu</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Passmode</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="4">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Máy khắc laser</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="txtIVDir">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="20" column="1">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Barcode</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="3">
         <widget class="QLabel" name="label_25">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PLC</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Mật khẩu leader</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="btnBrowseData">
          <property name="text">
           <string>Chọn</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="btnBrowseIV">
          <property name="text">
           <string>Chọn</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QPushButton" name="btnEnumCOM">
          <property name="text">
           <string>Liệt kê</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="txtAdminPassword">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="txtSuperPassword">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="txtLeaderPassword">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="2" colspan="2">
         <widget class="QLineEdit" name="txtPLCIP">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="2" colspan="2">
         <widget class="QSpinBox" name="spxPLCPort">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="12" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegCvWidth">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="13" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegStopper">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="14" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegPassMode">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="15" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegSWReady">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="16" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegTrigger">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="17" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegTriggerConfirm">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="18" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegStop">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="19" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegStopConfirm">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="20" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegBarCode">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="21" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegBarcodeConfirm">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="22" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegBarcodeData">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="23" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegBarcodeOK">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="24" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegBarcodeNG">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="25" column="2" colspan="2">
         <widget class="QLineEdit" name="txtRegPrintRes">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="27" column="2" colspan="2">
         <widget class="HxTableView" name="tbvStoppers">
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>24</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>24</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionMigration"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../Assets/Icons/icons.qrc">
     <normaloff>:/save_48px.png</normaloff>:/save_48px.png</iconset>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../Assets/Icons/icons.qrc">
     <normaloff>:/reset_48px.png</normaloff>:/reset_48px.png</iconset>
   </property>
   <property name="text">
    <string>Tải lại</string>
   </property>
  </action>
  <action name="actionMigration">
   <property name="icon">
    <iconset resource="../Assets/Icons/icons.qrc">
     <normaloff>:/database.png</normaloff>:/database.png</iconset>
   </property>
   <property name="text">
    <string>Migration</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HxTableView</class>
   <extends>QTableView</extends>
   <header>HxTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Assets/Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
